packages/fileeditor-extension/src/commands.ts

```diff
@@ -1247,9 +1246,11 @@ export namespace Commands {
       widget.title.caption = widget.title.label;
 
       // Get the fileType based on the mimetype to determine the icon
-      const fileType = toArray(app.docRegistry.fileTypes()).find(fileType => {
-        return mimetype ? fileType.mimeTypes.includes(mimetype) : undefined;
-      });
+      const fileType = Array.from(app.docRegistry.fileTypes()).find(
+        fileType => {
+          return mimetype ? fileType.mimeTypes.includes(mimetype) : undefined;
+        }
+      );
```

Comment:
Could we use `find` directly?

```suggestion
      const fileType = find(app.docRegistry.fileTypes(),
        fileType => {
          return mimetype ? fileType.mimeTypes.includes(mimetype) : undefined;
        }
      );
```